### PR TITLE
Advanced HTML and CSS lesson: Semantic HTML

### PR DESCRIPTION
**Pull request by devinprater**
Complete the following REQUIRED checkboxes:

-   [ ] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/theodinproject/blob/main/CONTRIBUTING.md)
-   [ ] The title of this PR follows the `location of change: brief description of change` format, e.g. `Intro to HTML and CSS lesson: Fix link text`

Complete the following checkboxes ONLY IF they are applicable to your PR. You can complete them later if they are not currently applicable:
-   [ ] I have previewed all lesson files included in this PR with the [Markdown preview tool](https://www.theodinproject.com/lessons/preview) to ensure the Markdown content is formatted correctly
-   [ ] I have ensured all lesson files included in this PR follow the [Layout Style Guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md)

<hr>

**1. Because:**

Closes #XXXXX

**2. This PR:**

**3. Additional Information:**

Fixes some info about screen readers, expands info on screen reader navigation, and fixes typos.

### CODE REVIEW

File: archive/html_css/semantic_html.md [hunk 1]
```diff
@@ -98,4 +98,6 @@ This section contains helpful links to other content. It isn’t required, so co
...
 * [Screen Reader Basics: VoiceOver](https://www.youtube.com/watch?v=5R-6WvAihms&list=PLNYkxOF6rcICWx0C9LVWWVqvHlYJyqw7g&index=8) and [Screen Reader Basics: NVDA](https://www.youtube.com/watch?v=Jao3s_CwdRU&list=PLNYkxOF6rcICWx0C9LVWWVqvHlYJyqw7g&index=9) go over some basics for getting started with both brands of screen readers. Both videos are worth watching regardless of which screen reader you choose/is available to you, as they contain some universal information as well.
+* ]Screen reader Basics: ChromeVox](https://www.youtube.com/watch?v=fpbIsN31hLM) Is a good place to start learning about the screen reader on ChromeBooks.
+* [Screen reader Basics: Orca](https://www.youtube.com/watch?v=UI76P-KPZec) is where you can learn about the screen reader on Linux desktops.
 * [ARIA Landmarks Example](https://www.w3.org/TR/wai-aria-practices/examples/landmarks/HTML5.html) lists the native HTML elements that define landmark regions as well as what their role is. If you check this resource out, be sure to enable the button toggles at the top of the page to see how the page is sectioned by landmarks and how it uses headings.
```
Comment (wise-king-sullyman) [inline archive/html_css/semantic_html.md:103]: I would undo the changes in this file, the archive directory is essentially just a graveyard for content once we've deprecated it for one reason or another.

**Review by wise-king-sullyman — CHANGES_REQUESTED** [review]
I like most of these changes but will need to check out the additional resources linked before I can approve.

In the meantime, I would like to see the changes in the archived file undone.

@thatblindgeye if you get a chance to take a look at this I would appreciate your feedback as well, especially with regards to the additional resources.

**Comment by wise-king-sullyman** [comment]
Additionally, @devinprater thank you for wanting to contribute, but please read our PR template and follow its instructions in any future PRs. Small things like the format we request for PR names make a big difference in maintainability for us.

File: advanced_html_css/accessibility/semantic_html.md [hunk 2]
```diff
@@ -98,4 +98,6 @@ This section contains helpful links to other content. It isn’t required, so co
...
 * [Screen Reader Basics: VoiceOver](https://www.youtube.com/watch?v=5R-6WvAihms&list=PLNYkxOF6rcICWx0C9LVWWVqvHlYJyqw7g&index=8) and [Screen Reader Basics: NVDA](https://www.youtube.com/watch?v=Jao3s_CwdRU&list=PLNYkxOF6rcICWx0C9LVWWVqvHlYJyqw7g&index=9) go over some basics for getting started with both brands of screen readers. Both videos are worth watching regardless of which screen reader you choose/is available to you, as they contain some universal information as well.
+* [screen reader basics: Orca](https://www.youtube.com/watch?v=UI76P-KPZec) is a good video to get you started using Orca, the Linux GUI screen reader.
+* [Screen reader basics: ChromeVox](https://www.youtube.com/watch?v=fpbIsN31hLM) is a starting place for learning to use ChromeVox, the screen reader on ChromeBooks.
```
Comment (thatblindgeye) [inline advanced_html_css/accessibility/semantic_html.md:102]: Nitpick: capitalization in the link titles here

File: archive/html_css/semantic_html.md [hunk 3]
```diff
@@ -75,7 +75,7 @@ Headings are the `<h1>` through `<h6>` elements, and like the name implies, thes
 * `<nav>`
```
Comment (thatblindgeye) [inline archive/html_css/semantic_html.md:75]: I don't think we really need to update the archived files. @wise-king-sullyman any opinion?

**Review by thatblindgeye — COMMENTED** [review]
Thanks for the changes! Below are some file specific comments.

File: archive/html_css/semantic_html.md (hunk 3 above)
Comment (wise-king-sullyman) [reply archive/html_css/semantic_html.md:75]: Agree

**Comment by thatblindgeye** [comment]
@devinprater Just wanted to check in to see if you've seen the request changes above? In regards to the second comment, the archived file should be reverted to what it previously was. I don't think updating archived files serves any real purpose.

**Comment by devinprater** [comment]
Yeah, sorry. I don't know how to revert the file.
Devin Prater
***@***.***

**Comment by wise-king-sullyman** [comment]
@devinprater since the changes were all done under a single commit the most straightforward way will just be to undo the changes you made manually, using the version of the file that is currently in our repo as a reference.

If you'd like you can do it with `git revert` as well and just revert that one file, but you'll have to do a bit more research first. "git revert single file" as a google search should return good results.

File: archive/html_css/semantic_html.md [hunk 4]
```diff
@@ -75,7 +75,7 @@ Headings are the `<h1>` through `<h6>` elements, and like the name implies, thes
...
 * `<section>`
 
-By properly using landmarks and headings, you provide users of assistive technologies a more operable and understandbale page: not only can screen readers navigate a page via landmarks and headings by opening a menu in their screen reader, but these elements also have their roles announced to provide additional context.
+By properly using landmarks and headings, you provide users of assistive technologies a more operable and understandbale page: not only can screen readers navigate a page via landmarks and headings by using quick navigation keyboard commands (or opening a menu in their screen reader), but these elements also have their roles announced to provide additional context.
```
Comment (zachmmeyer) [inline archive/html_css/semantic_html.md:78]:
```suggestion
By properly using landmarks and headings, you provide users of assistive technologies a more operable and understandbale page: not only can screen readers navigate a page via landmarks and headings by opening a menu in their screen reader, but these elements also have their roles announced to provide additional context.
```

File: archive/html_css/semantic_html.md [hunk 5]
```diff
@@ -98,4 +98,4 @@ This section contains helpful links to other content. It isn’t required, so co
...
-* [ARIA Landmarks Example](https://www.w3.org/TR/wai-aria-practices/examples/landmarks/HTML5.html) lists the native HTML elements that define landmark regions as well as what their role is. If you check this resource out, be sure to enable the button toggles at the top of the page to see how the page is sectioned by landmarks and how it uses headings.
\ No newline at end of file
+* [ARIA Landmarks Example](https://www.w3.org/TR/wai-aria-practices/examples/landmarks/HTML5.html) lists the native HTML elements that define landmark regions as well as what their role is. If you check this resource out, be sure to enable the button toggles at the top of the page to see how the page is sectioned by landmarks and how it uses headings.
```
Comment (zachmmeyer) [inline archive/html_css/semantic_html.md:101]:
```suggestion
* [ARIA Landmarks Example](https://www.w3.org/TR/wai-aria-practices/examples/landmarks/HTML5.html) lists the native HTML elements that define landmark regions as well as what their role is. If you check this resource out, be sure to enable the button toggles at the top of the page to see how the page is sectioned by landmarks and how it uses headings.
```

**Review by zachmmeyer — CHANGES_REQUESTED** [review]

**Review by zachmmeyer — APPROVED** [review]
